crates/modelardb_server/src/metadata/mod.rs

```diff
@@ -1105,6 +1406,494 @@ mod tests {
+        let compressed_file = CompressedFile::new(Uuid::new_v4(), 100, 200, 37.0, 73.0);
+        assert!(
+            create_metadata_manager_with_named_model_table_and_save_files(
+                "table_model",
+                &[compressed_file]
+            )
+            .await
+            .is_err()
+        );
+    }
+
+    #[tokio::test]
+    async fn test_save_compressed_file_with_swaped_timestamps_for_column_in_existing_model_table() {
+        let compressed_file = CompressedFile::new(Uuid::new_v4(), 200, 100, 37.0, 73.0);
+        assert!(
+            create_metadata_manager_with_named_model_table_and_save_files(
+                "model_table",
+                &[compressed_file]
+            )
+            .await
+            .is_err()
+        );
+    }
+
+    #[tokio::test]
+    async fn test_save_compressed_file_with_swaped_values_for_column_in_existing_model_table() {
+        let compressed_file = CompressedFile::new(Uuid::new_v4(), 100, 200, 73.0, 37.0);
+        assert!(
+            create_metadata_manager_with_named_model_table_and_save_files(
+                "model_table",
+                &[compressed_file]
+            )
+            .await
+            .is_err()
+        );
+    }
+
+    async fn create_metadata_manager_with_named_model_table_and_save_files(
+        model_table_name: &str,
+        compressed_files: &[CompressedFile],
+    ) -> Result<MetadataManager> {
+        let temp_dir = tempfile::tempdir().unwrap();
+        let metadata_manager = create_metadata_manager_and_save_model_table(temp_dir.path()).await;
+
+        for compressed_file in compressed_files {
+            metadata_manager
+                .save_compressed_file_for_column_in_model_table(
+                    model_table_name,
+                    1,
+                    compressed_file,
+                )
+                .await?;
+        }
+
+        Ok(metadata_manager)
+    }
+
+    #[tokio::test]
+    async fn test_replace_compressed_files_for_column_in_model_table_with_files_to_delete() {
+        let compressed_files = SEVEN_COMPRESSED_FILES.to_vec();
+        let uuids_of_files_to_delete = &[
+            compressed_files[0].name,
+            compressed_files[1].name,
+            compressed_files[2].name,
+        ];
+        let returned_files =
+            create_metadata_manager_with_named_model_table_save_files_delete_and_get(
+                &compressed_files,
+                uuids_of_files_to_delete,
+                None,
+            )
+            .await;
+
+        assert_eq!(4, returned_files.len());
+        assert_eq!(compressed_files[3].name, returned_files[0]);
+        assert_eq!(compressed_files[4].name, returned_files[1]);
+        assert_eq!(compressed_files[5].name, returned_files[2]);
+        assert_eq!(compressed_files[6].name, returned_files[3]);
+    }
+
+    #[tokio::test]
+    async fn test_replace_compressed_files_for_column_in_model_table_with_files_to_replace() {
```

Comment:
I am not sure i understand this test name. It makes it sound like the compressed files are replaced with the files to replace. Maybe rename it to something like `test_replace_compressed_files_for_column_in_model_table_with_replacement_file`

Reply:
Fixed using suggested name.